unit test TestSyncerSimple fails

FAIL:TestSyncerSimple

## Bug Report

Please answer these questions before submitting your issue. Thanks!

1. What did you do?
If possible, provide a recipe for reproducing the error.

https://internal.pingcap.net/idc-jenkins/blue/organizations/jenkins/tidb_ghpr_check_2/detail/tidb_ghpr_check_2/23932/pipeline/

https://internal.pingcap.net/idc-jenkins/blue/organizations/jenkins/tidb_ghpr_unit_test/detail/tidb_ghpr_unit_test/23968/pipeline/

```


[2020-02-13T04:01:46.004Z] --- FAIL: TestSyncerSimple (12.90s)

[2020-02-13T04:01:46.004Z]     syncer_test.go:119: get udpate version failed

[2020-02-13T04:01:46.004Z] FAIL
```

2. What did you expect to see?



3. What did you see instead?



4. What version of TiDB are you using (`tidb-server -V` or run `select tidb_version();` on TiDB)?

Another new error message.
```
[2020-03-30T09:33:34.467Z] === RUN   TestSyncerSimple

[2020-03-30T09:33:34.467Z] {"level":"warn","ts":"2020-03-30T17:32:53.734+0800","caller":"clientv3/retry_interceptor.go:61","msg":"retrying of unary invoker failed","target":"endpoint://client-5765beae-6afa-43fb-8ca3-3a359e382572/localhost:46823190510607748800","attempt":0,"error":"rpc error: code = Canceled desc = context canceled"}

[2020-03-30T09:33:34.467Z] {"level":"warn","ts":"2020-03-30T17:32:53.734+0800","caller":"clientv3/retry_interceptor.go:61","msg":"retrying of unary invoker failed","target":"endpoint://client-5765beae-6afa-43fb-8ca3-3a359e382572/localhost:46823190510607748800","attempt":0,"error":"rpc error: code = Canceled desc = grpc: the client connection is closing"}

[2020-03-30T09:33:34.467Z] --- FAIL: TestSyncerSimple (0.36s)

[2020-03-30T09:33:34.467Z]     syncer_test.go:244: resp key session_ttl_key kvs [key:"session_ttl_key" create_revision:10 mod_revision:10 version:1 value:"session_ttl_val" lease:7015606748117666343 ] length is != 0

[2020-03-30T09:33:34.467Z] FAIL

[2020-03-30T09:33:34.467Z] FAIL	github.com/pingcap/tidb/ddl/util	0.384s
```

@bb7133 PTAL

--- a/ddl/util/syncer_test.go
+++ b/ddl/util/syncer_test.go
@@ -116,7 +116,7 @@ func TestSyncerSimple(t *testing.T) {
 				t.Fatalf("get chan events count less than 1")
 			}
 			checkRespKV(t, 1, DDLGlobalSchemaVersion, fmt.Sprintf("%v", currentVer), resp.Events[0].Kv)
-		case <-time.After(100 * time.Millisecond):
+		case <-time.After(3 * time.Second):
 			t.Fatalf("get udpate version failed")
 		}
 	}()

